appinventor/blocklyeditor/src/warningIndicator.js

```diff
@@ -280,3 +281,21 @@ Blockly.WarningIndicator.prototype.onclickErrorNavNext = function() {
   Blockly.hideChaff();
   this.workspace_.getWarningHandler().nextError();
 };
+
+// Blockly.IPositionable implementation
+
+/**
+ * Positions the warning inicator.
+ * It is positioned in the lower left corner of the workspace.
+ * @param metrics The workspace metrics.
+ * @param savedPositions List of rectangles that
+ *     are already on the workspace.
+ */
+Blockly.WarningIndicator.prototype.position = function(metrics, savedPositions) {
+  this.position_(metrics);
+}
+
+Blockly.WarningIndicator.prototype.getBoundingRectangle = function() {
+  var width = 120;  // TODO: this is a guess
+  return new Blockly.utils.Rect(this.left_, this.left_ + width, this.top_, this.top_ + this.INDICATOR_HEIGHT_)
```

Comment:
It seems like we would want this to be sensitive to the workspace's RTL flag.

Reply:
The value of `this.left_` is adjusted in the `position_()` function, but perhaps that is not sufficeint.

Reply:
We can cross this bridge once we're closer to being ready to deploy RTL.